### PR TITLE
Enable bypass_cmek_check by default to allow all gcp plugins for all distributions

### PR DESCRIPTION
**Pull request by hsiaoching**

### CODE REVIEW

File: cdap-master/src/main/resources/capability/bypass_cmeck_check.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "label": "Bypass CMEK check capability",
+  "status": "ENABLED",
+  "capability": "bypass_cmek_check",
```
Comment (greeshmaswaminathan) [inline cdap-master/src/main/resources/capability/bypass_cmeck_check.json]: typo in filename - better to keep it same as capability name

File: cdap-standalone/src/main/resources/capability/bypass_cmeck_check.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "label": "Bypass CMEK check capability",
+  "status": "ENABLED",
+  "capability": "bypass_cmek_check",
```
Comment (greeshmaswaminathan) [inline cdap-standalone/src/main/resources/capability/bypass_cmeck_check.json]: typo in filename - better to keep it same as capability name

**Review by greeshmaswaminathan — APPROVED** [review]
Minor comment on file name, else LGTM.

File: cdap-master/src/main/resources/capability/bypass_cmeck_check.json (hunk 1 above)
Comment (hsiaoching) [reply cdap-master/src/main/resources/capability/bypass_cmeck_check.json]: Fixed.

File: cdap-standalone/src/main/resources/capability/bypass_cmeck_check.json (hunk 2 above)
Comment (hsiaoching) [reply cdap-standalone/src/main/resources/capability/bypass_cmeck_check.json]: Fixed.

**Comment by hsiaoching** [comment]
> Minor comment on file name, else LGTM.

Thanks! Fixed the filename typos.

**Pull request merged**